csrc/runtime/executor_abstract.h

```diff
@@ -0,0 +1,49 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <fusion.h>
+#include <runtime/executor_kernel_arg.h>
+#include <runtime/executor_params.h>
+#include <runtime/executor_utils.h>
+#include <scheduler/scheduler_types.h>
+
+namespace nvfuser {
+
+class ExecutorAbstract : public PolymorphicBase, public NonCopyable {
+ public:
+  //   // NVF_API was added for nvfuser_extension. See examples/sinh_extension.
```

Comment:
```suggestion
```
as long as CI passes without NVF_API

Reply:
I need to recheck this, I'm guessing the comment can be removed, as I imagine the extension would use KernelExecutor not ExecutorAbstract.